awscli/urls

```diff
@@ -0,0 +1 @@
+https://awscli.amazonaws.com/${code}.${ext}
```

Comment:
I would prefer os-mapping instead of using `${code}` as this requires hardcoding in the aws cli commandlet.

Reply:
```
windows=AWSCLIV2
mac=AWSCLIV2
*=awscli-exe-linux-x86_64
```